### ISSUE TITLE
Suggestion: Add Save state (and possibly game save) form to the bug report

### ISSUE THREAD

**Issue by davidhedlund** [hrydgard/ppsspp#18619]
### Game or games this happens in

*

### What area of the game / PPSSPP

*

### What should happen

Example, the bug in this game occurs at the end of the game, so the developer are dependent of the save state:
* https://github.com/hrydgard/ppsspp/issues/14958

### Platform

Android

### Mobile device model or graphics card (GPU)

*

### PPSSPP version affected

*

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
That's a good idea, will do.

**Comment by davidhedlund** [comment]
> That's a good idea, will do.
> 
> Although, I don't know how to actually create an "attach file" field to a template... There doesn't seem to be a "File" type attachment: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
> 
> Though, we could of course add some text...

Check out Yuzu's [bug report template](https://github.com/yuzu-emu/yuzu/issues/new?assignees=&labels=&projects=&template=bug_report.yml):

![image](https://github.com/hrydgard/ppsspp/assets/6416073/884cf588-67ad-4003-9a66-8305ce0d47c8)

Perhaps you can ask them how to set it up if you don't find anything in the GitHub documentation.

**Comment by davidhedlund** [comment]
Also, GitHub accepts only the most common file formats. So I guess that some text should be added that "the ppst file has to be zipped with the JPEG file".

**Comment by hrydgard** [comment]
Ah, that's pretty good. I'll mimic what they do.

**Comment by davidhedlund** [comment]
> Ah, that's pretty good. I'll mimic what they do.

Of course, you can add a "Log file" form field as well while you are doing this. Up to you.
